lib/ansible/parsing/vault/__init__.py

```diff
@@ -319,10 +319,9 @@ def bytes(self):
         return None
 
     def load(self):
-        self._bytes = self.read_file(self.filename, self.loader)
+        self._bytes = self.read_file(self.filename)
 
-    @staticmethod
-    def read_file(filename, loader):
+    def read_file(self, filename):
```

Comment:
This is a bad change.  self is not used inside of this method at all.  I'm not sure which pylint warning you're seeing but the answer for it is probably to make this a toplevel function, private to this python module, not a regular method.